### PR TITLE
Completed metric

### PR DESCRIPTION
**Pull request by jyyy03**
Metric does not involve pytorch to tensorflow conversion. It's simply copy and paste. 
See metric_test.ipynb for tests.

- TODO: `main` function is waiting for arguments parser in `evaluate.py`

### CODE REVIEW

**Review by wanghungjen — APPROVED** [review]
Great!! Thank you so much :))
